### PR TITLE
Add The text "My Changes " To the Artical

### PR DESCRIPTION
**Pull request by tofiqueahmedkhan**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ ] My pull request targets the `master` branch of freeCodeCamp.
- [ ] None of my changes are plagiarized from another source without proper attribution.
- [ ] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/english/bootstrap/index.md [hunk 1]
```diff
@@ -83,3 +83,7 @@ themes that build on the Bootstrap framework to provide a more customized and st
 [HTML linter for projects using Bootstrap](https://github.com/twbs/bootlint)
 [Design elements and code snippets for Bootstrap](https://bootsnipp.com/)
 [Code, theme, and add-on resources for Bootstrap](http://expo.getbootstrap.com/resources/)
+
+My Changes.
```
Comment (Manish-Giri) [inline guide/english/bootstrap/index.md]: Delete "My Changes."

File: guide/english/bootstrap/index.md [hunk 2]
```diff
@@ -83,3 +83,7 @@ themes that build on the Bootstrap framework to provide a more customized and st
...
+
+Bootstrap is designed to be mobile first, meaning that the classes are designed such that we can begin by targeting mobile device screens first and then work upwards to larger screen sizes.
```
Comment (Manish-Giri) [inline guide/english/bootstrap/index.md]: Add your changes to the "Boostrap Features" section as a list item.

**Review by Manish-Giri — CHANGES_REQUESTED** [review]
Suggested changes to be addressed.

**Comment by ArielLeslie** [comment]
Stop putting the words 'My Changes' in random places.

**Review by Manish-Giri — COMMENTED** [review]
Hi @tofiqueahmedkhan,

As there has been no response from your end for a while, I am closing this PR.

Feel free to re-open it if you've addressed the suggested changes.

**Pull request closed without merging**